composer.json

```diff
@@ -57,7 +57,7 @@
     "cweagans/composer-patches": "~1.0",
     "drupal/core-recommended": "^10",
     "drupal/semver_example": "2.3.0",
-    "phpunit/phpunit": ">=9",
+    "phpunit/phpunit": "^9",
```

Comment:
What led to forbidding phpunit 10? Its only a dev dependency so not a big deal - just curious.

Reply:
Appveyor started running on PHPUnit 10 and broke. I figure we can fix this as follow-on work.